Logging: "after all" hook in "Logging" failed

This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: 413c8e8afc303bc1c29d13ed650cc7c16c635ac3
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/8c5d3f91-cdc5-4d63-ad7f-c70b1851c12c), [Sponge](http://sponge2/8c5d3f91-cdc5-4d63-ad7f-c70b1851c12c)
status: failed
<details><summary>Test output</summary><br><pre>No such object: nodejs-logging-system-test-1630525713073-98e62e27ef88/cloudaudit.googleapis.com/policy/2021/09/02/09:00:00_09:59:59_S27.json
Error: No such object: nodejs-logging-system-test-1630525713073-98e62e27ef88/cloudaudit.googleapis.com/policy/2021/09/02/09:00:00_09:59:59_S27.json
    at new ApiError (node_modules/@google-cloud/common/build/src/util.js:73:15)
    at Util.parseHttpRespBody (node_modules/@google-cloud/common/build/src/util.js:208:38)
    at Util.handleResp (node_modules/@google-cloud/common/build/src/util.js:149:117)
    at /workspace/node_modules/@google-cloud/common/build/src/util.js:477:22
    at onResponse (node_modules/retry-request/index.js:228:7)
    at /workspace/node_modules/teeny-request/build/src/index.js:226:13
        -> /workspace/node_modules/teeny-request/src/index.ts:333:11
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)</pre></details>

Looking into the logged error: `npm ERR! code ELIFECYCLE` and the [stackoverflow article](https://stackoverflow.com/questions/42308879/how-to-solve-npm-error-npm-err-code-elifecycle), the problem is not in the test but in the environment.

--- a/system-test/logging.ts
+++ b/system-test/logging.ts
@@ -118,7 +118,14 @@ describe('Logging', () => {
             }
             return getDateFromGeneratedName(name) < oneHourAgo;
           })
-          .map(o => o.delete())
+          .map(o => {
+            const name = o.name || o.id;
+            try {
+              o.delete();
+            } catch (err) {
+              console.warn(`failed to delete "${name}": ${err}`);
+            }
+          })
       );
     }
   });

